src/include/platform/ThreadStackManager.h

```diff
@@ -241,6 +242,11 @@ inline CHIP_ERROR ThreadStackManager::InitThreadStack()
     return static_cast<ImplClass *>(this)->_InitThreadStack();
 }
 
+inline void ThreadStackManager::DeinitThreadStack()
```

Comment:
Should we separate this function to two functions `StopThreadTask()`and  `DeinitThreadStack()` as we already have `StartThreadTask()` and `InitThreadStack()`?

Reply:
Sure, done